src/bin/nydus-image/main.rs

```diff
@@ -720,8 +720,16 @@ fn prepare_cmd_args(bti_string: &'static str) -> App {
             .arg(
                 Arg::new("output")
                     .long("output")
-                    .help("path for output tar file")
-                    .required(true),
+                    .default_value("/dev/stdout")
+                    .help("Path for output tar file, if not configured, it will default to STDOUT.")
+                    .required(false),
+            )
+            .arg(
+                Arg::new("untar")
```

Comment:
A smoke test is required for new option.
Besides update doc please.

Reply:
@ccx1024cc Unit testing is complete and now changed to WIP.